battery_monitor/src/battery_monitor.cpp

```diff
@@ -171,18 +197,27 @@ BatteryMonitor::BatteryMonitor(ros::NodeHandle &nodeHandle)
           diagnostic_tools::PeriodicMessageStatusParams{}
               .min_acceptable_period(minBatteryLeakCheckPeriod)
               .max_acceptable_period(maxBatteryLeakCheckPeriod)
-              .abnormal_diagnostic({diagnostic_tools::Diagnostic::WARN,
-                                    health_monitor::ReportFault::BATTERY_INFO_STALE})));
+              .abnormal_diagnostic(
+              {
+                diagnostic_tools::Diagnostic::WARN,
+                health_monitor::ReportFault::BATTERY_INFO_STALE
+              }
+)));
```

Comment:
@patapufate meta: hmm, checks' construction is not particularly easy to read. Perhaps we should split things up a bit?

Reply:
How do you think it could be the best way to split checks?

Reply:
Lambdas and param structs could be put together right above. For instance, 

```c++
diagnostic_tools::PeriodicMessageStatusParams params{};
params.min_acceptable_period(minBatteryLeakCheckPeriod)
params.max_acceptable_period(maxBatteryLeakCheckPeriod)
diagnostic_tools::Diagnostic diag{
    diagnostic_tools::Diagnostic::WARN, 
    health_monitor::ReportFault::BATTERY_INFO_STALE};
params.abnormal_diagnostic(diag);
// ...
```

Or similar. It should be easy to grok, that's the only consideration.

Reply:
Great. Thanks.
Done